diff --git a/out/AppUpdater.d.ts b/out/AppUpdater.d.ts
index 5979c3019023980f43e16759fa090e365504a17e..57f02a3f7bb491d56759090b53ac5136fef95b78 100644
--- a/out/AppUpdater.d.ts
+++ b/out/AppUpdater.d.ts
@@ -141,16 +141,7 @@ export declare abstract class AppUpdater extends AppUpdater_base {
      */
     get isUpdateSupported(): VerifyUpdateSupport;
     set isUpdateSupported(value: VerifyUpdateSupport);
-    protected _isUserWithinRollout: VerifyUpdateSupport;
-    /**
-     * Allows developer to override default logic for determining if the user is below the rollout threshold.
-     * The default logic compares the staging percentage with numerical representation of user ID.
-     * An override can define custom logic, or bypass it if needed.
-     */
-    get isUserWithinRollout(): VerifyUpdateSupport;
-    set isUserWithinRollout(value: VerifyUpdateSupport);
     private clientPromise;
-    protected readonly stagingUserIdPromise: Lazy<string>;
     private checkForUpdatesPromise;
     private downloadPromise;
     protected readonly app: AppAdapter;
@@ -170,7 +161,6 @@ export declare abstract class AppUpdater extends AppUpdater_base {
     isUpdaterActive(): boolean;
     checkForUpdatesAndNotify(downloadNotification?: DownloadNotification): Promise<UpdateCheckResult | null>;
     private static formatDownloadNotification;
-    private isStagingMatch;
     private computeFinalHeaders;
     private isUpdateAvailable;
     private checkIfUpdateSupported;
@@ -200,7 +190,6 @@ export declare abstract class AppUpdater extends AppUpdater_base {
     abstract quitAndInstall(isSilent?: boolean, isForceRunAfter?: boolean): void;
     private loadUpdateConfig;
     private computeRequestHeaders;
-    private getOrCreateStagingUserId;
     private getOrCreateDownloadHelper;
     protected executeDownload(taskOptions: DownloadExecutorTask): Promise<Array<string>>;
     protected differentialDownloadInstaller(fileInfo: ResolvedUpdateFileInfo, downloadUpdateOptions: DownloadUpdateOptions, installerPath: string, provider: Provider<any>, oldInstallerFileName: string): Promise<boolean>;
diff --git a/out/AppUpdater.js b/out/AppUpdater.js
index 7358a8f0efca83c0e05186fa51f154592b66c8dd..64696e8808a6670905f70c8b73dadfdfb54dca49 100644
--- a/out/AppUpdater.js
+++ b/out/AppUpdater.js
@@ -2,7 +2,6 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.NoOpLogger = exports.AppUpdater = void 0;
 const builder_util_runtime_1 = require("builder-util-runtime");
-const crypto_1 = require("crypto");
 const os_1 = require("os");
 const events_1 = require("events");
 const fs_extra_1 = require("fs-extra");
@@ -87,19 +86,6 @@ class AppUpdater extends events_1.EventEmitter {
             this._isUpdateSupported = value;
         }
     }
-    /**
-     * Allows developer to override default logic for determining if the user is below the rollout threshold.
-     * The default logic compares the staging percentage with numerical representation of user ID.
-     * An override can define custom logic, or bypass it if needed.
-     */
-    get isUserWithinRollout() {
-        return this._isUserWithinRollout;
-    }
-    set isUserWithinRollout(value) {
-        if (value) {
-            this._isUserWithinRollout = value;
-        }
-    }
     constructor(options, app) {
         super();
         /**
@@ -184,9 +170,7 @@ class AppUpdater extends events_1.EventEmitter {
         this.signals = new types_1.UpdaterSignal(this);
         this._appUpdateConfigPath = null;
         this._isUpdateSupported = updateInfo => this.checkIfUpdateSupported(updateInfo);
-        this._isUserWithinRollout = updateInfo => this.isStagingMatch(updateInfo);
         this.clientPromise = null;
-        this.stagingUserIdPromise = new lazy_val_1.Lazy(() => this.getOrCreateStagingUserId());
         // public, allow to read old config for anyone
         /** @internal */
         this.configOnDisk = new lazy_val_1.Lazy(() => this.loadUpdateConfig());
@@ -311,25 +295,6 @@ class AppUpdater extends events_1.EventEmitter {
         };
         return downloadNotification;
     }
-    async isStagingMatch(updateInfo) {
-        const rawStagingPercentage = updateInfo.stagingPercentage;
-        let stagingPercentage = rawStagingPercentage;
-        if (stagingPercentage == null) {
-            return true;
-        }
-        stagingPercentage = parseInt(stagingPercentage, 10);
-        if (isNaN(stagingPercentage)) {
-            this._logger.warn(`Staging percentage is NaN: ${rawStagingPercentage}`);
-            return true;
-        }
-        // convert from user 0-100 to internal 0-1
-        stagingPercentage = stagingPercentage / 100;
-        const stagingUserId = await this.stagingUserIdPromise.value;
-        const val = builder_util_runtime_1.UUID.parse(stagingUserId).readUInt32BE(12);
-        const percentage = val / 0xffffffff;
-        this._logger.info(`Staging percentage: ${stagingPercentage}, percentage: ${percentage}, user id: ${stagingUserId}`);
-        return percentage < stagingPercentage;
-    }
     computeFinalHeaders(headers) {
         if (this.requestHeaders != null) {
             Object.assign(headers, this.requestHeaders);
@@ -348,10 +313,6 @@ class AppUpdater extends events_1.EventEmitter {
         if (!(await Promise.resolve(this.isUpdateSupported(updateInfo)))) {
             return false;
         }
-        const isUserWithinRollout = await Promise.resolve(this.isUserWithinRollout(updateInfo));
-        if (!isUserWithinRollout) {
-            return false;
-        }
         // https://github.com/electron-userland/electron-builder/pull/3111#issuecomment-405033227
         // https://github.com/electron-userland/electron-builder/pull/3111#issuecomment-405030797
         const isLatestVersionNewer = (0, semver_1.gt)(latestVersion, currentVersion);
@@ -383,8 +344,7 @@ class AppUpdater extends events_1.EventEmitter {
             this.clientPromise = this.configOnDisk.value.then(it => (0, providerFactory_1.createClient)(it, this, this.createProviderRuntimeOptions()));
         }
         const client = await this.clientPromise;
-        const stagingUserId = await this.stagingUserIdPromise.value;
-        client.setRequestHeaders(this.computeFinalHeaders({ "x-user-staging-id": stagingUserId }));
+        client.setRequestHeaders(this.computeFinalHeaders({}));
         return {
             info: await client.getLatestVersion(),
             provider: client,
@@ -498,32 +458,6 @@ class AppUpdater extends events_1.EventEmitter {
         }
         return this.computeFinalHeaders({ accept: "*/*" });
     }
-    async getOrCreateStagingUserId() {
-        const file = path.join(this.app.userDataPath, ".updaterId");
-        try {
-            const id = await (0, fs_extra_1.readFile)(file, "utf-8");
-            if (builder_util_runtime_1.UUID.check(id)) {
-                return id;
-            }
-            else {
-                this._logger.warn(`Staging user id file exists, but content was invalid: ${id}`);
-            }
-        }
-        catch (e) {
-            if (e.code !== "ENOENT") {
-                this._logger.warn(`Couldn't read staging user ID, creating a blank one: ${e}`);
-            }
-        }
-        const id = builder_util_runtime_1.UUID.v5((0, crypto_1.randomBytes)(4096), builder_util_runtime_1.UUID.OID);
-        this._logger.info(`Generated new staging user ID: ${id}`);
-        try {
-            await (0, fs_extra_1.outputFile)(file, id);
-        }
-        catch (e) {
-            this._logger.warn(`Couldn't write out staging user ID: ${e}`);
-        }
-        return id;
-    }
     /** @internal */
     get isAddNoCacheQuery() {
         const headers = this.requestHeaders;
@@ -731,4 +665,3 @@ class NoOpLogger {
     }
 }
 exports.NoOpLogger = NoOpLogger;
-//# sourceMappingURL=AppUpdater.js.map
\ No newline at end of file
diff --git a/out/AppUpdater.js.map b/out/AppUpdater.js.map
deleted file mode 100644
index aa0baab8e92106c4473321b2373ca9935efde902..0000000000000000000000000000000000000000
